ocs_ci/helpers/keyrotation_helper.py

```diff
@@ -408,3 +408,120 @@ def wait_till_keyrotation(self, device_handle):
             assert False
 
         return True
+
+
+def enable_key_rotation():
+    """
+    Enable Key rotation and verify its status for Noobaa and Storage cluster.
+
+    """
+    osd_keyrotation = OSDKeyrotation()
+    noobaa_keyrotation = NoobaaKeyrotation()
+    osd_keyrotation.enable_keyrotation()
+    noobaa_keyrotation.enable_keyrotation()
+
+    assert (
+        osd_keyrotation.is_keyrotation_enable()
+    ), "Encryption Key rotation is not enabled for OSDs"
+    assert (
+        noobaa_keyrotation.is_keyrotation_enable
+    ), "Keyrotation is not enabled in the storagecluster object."
+    assert (
+        noobaa_keyrotation.is_noobaa_keyrotation_enable
+    ), "Keyrotation is not enabled in the noobaa object."
+
+
+def set_key_rotation_time(value):
+    """
+    This function will edit the storage cluster and add the value in key rotation schedule
+    Args:
+         value (str): number of minutes
+
+    """
+    osd_keyrotation = OSDKeyrotation()
+    schedule = f"*/{value} * * * *"
+    osd_keyrotation.set_keyrotation_schedule(schedule)
+
+
+def verify_key_rotation_time(schedule):
```

Comment:
isn;t it same as https://github.com/paraggit/ocs-ci/blob/aeeaee724e9453ffd586616978f21bb4da5c95da/ocs_ci/helpers/keyrotation_helper.py#L331

Reply:
No they are not verify the time we set for key rotation.

Reply:
Even we can't reuse this function for next step to verify new key generated or not. Why because they are just verifying for OSDs [for this also we need to capture old keys ans pass them as argument]  and not checking for  noobaa. We are verifying for Noobaa volume, Noobaa backed and OSDs.